sormas-app/app/src/main/res/layout/dialog_exposure_read_layout.xml

```diff
@@ -163,6 +163,11 @@
 
                 </LinearLayout>
 
+                <de.symeda.sormas.app.component.controls.ControlTextReadField
```

Comment:
this has nothing to do with your task, please do a proper merge with the development branch